css/grayscale.css

```diff
@@ -176,6 +177,24 @@ a:focus {
     font-family: 'Dancing Script';
 }
 
+.bg-aboutUs {
+    display: table;
+    width: 100%;
+    height: auto;
+    padding: 100px 0;
+    text-align: center;
+    color: #fff;
+    background: url(../img/bk_gnd.jpg) no-repeat bottom center scroll;
+    background-color: #000;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    background-size: cover;
+    -o-background-size: cover;
+    -webkit-filter:  hue-rotate(90deg); /* Chrome, Safari, Opera */
```

Comment:
instead of hue rotate filter, try using grayscale filter or some other filter to make the text look okay http://www.w3schools.com/cssref/css3_pr_filter.asp